src/Sets/SimpleSparsePolynomialZonotope.jl

```diff
@@ -329,16 +329,18 @@ The result uses dense arrays irrespective of the array type of `S`.
 
 Let `G` be the generator matrix and `E` the exponent matrix of `S`. The following simplifications are performed:
 
-- Zero columns in `G` and the corresponding columns in `E` are eliminated.
+- Zero columns in `G` and the corresponding columns in `E` are removed.
 - For zero columns in `E`, the corresponding column in `G` is summed to the center.
 - Repeated columns in `E` are grouped together by summing the corresponding columns in `G`.
 """
 function remove_redundant_generators(S::SimpleSparsePolynomialZonotope)
 
-    c = center(S)
-    G = genmat(S)
-    E = expmat(S)
+    c, G, E = _compat(center(S), genmat(S), expmat(S))
 
+    return SimpleSparsePolynomialZonotope(c, G, E)
+end
+
+function _compat(c, G, E)
```

Comment:
It prefer descriptive names, even if they are long. For instance `_remove_redundant_generators_SPZ`.

Reply:
what about just `_remove_redundant_generators`? The reason for this being that the same helper is used both for SPZ and SSPZ

Reply:
The problem is that there may be other such helper methods and since there are no type restrictions, it may be confused. Maybe `_remove_redundant_generators_polyzono`?

Reply:
But I am fine with `_remove_redundant_generators` too if it works.